pyproject.toml

```diff
@@ -16,7 +16,7 @@ classifiers = [
   "License :: OSI Approved :: MIT License",
   "Operating System :: OS Independent",
 ]
-requires-python = ">=3.8"
+requires-python = ">=3.11,<3.13"
```

Comment:
we shouldn't set upper bounds here

- [guidance](https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#requires-python-upper-bounds)
- [more context](https://iscinumpy.dev/post/bound-version-constraints/#pinning-the-python-version-is-special)

Reply:
ah okay that makes sense! this is now updated @ejm714